pic_database.php

```diff
@@ -0,0 +1,61 @@
+<?php
+	include('config/config.php');
+	include('include/db_query.php');
+
+	// InsertPic('Wes Hicks', '2018-05-25', 'TestPic', 'pictures/0f191f-wes-hicks-464614-unsplash.jpg', 'flavortext');
+	$pic = GetPic($_REQUEST['picID']);
+	//var_Dump($pic);
+	DisplayPic($pic);
+	// echo"<img src='pictures/0f191f-wes-hicks-464614-unsplash.jpg'alt='".$pic['flavor']."'>";
+
+
+	function InsertPic($photographer, $date, $title, $link, $flavor){
+			$result = dbQuery("
+				INSERT INTO pics (photographer, date, title, link, flavor)
+				VALUES('$photographer', '$date', '$title', '$link', '$flavor')
+			")->fetch();
+		}
+
+		function GetPic($picID){
+			$result = dbQuery("
+				SELECT *
+				FROM pics
+				WHERE picID = $picID
+			")->fetch();
+			return $result;
+		}
+
+		function DeletePic($picID){
+			$result = dbQuery("
+				DELETE FROM pics
+				WHERE postID = $picID
+			")->fetch();
+		}
+
+		function GetAllPics($picID){
+			$result = dbQuery("
+				SELECT *
+				FROM pics
+			")->fetch();
+			return $result;
+		}
+
+		function DisplayPic($pic){
```

Comment:
Depends on how you want to use this, but sometimes it's better to have functions like this take a $picID instead of a $pic. Then you just call `GetPic` from within this function. That makes it more complete and easier to call from elsewhere.